### PR TITLE
[GL-3755] Metric tile documentation

### PR DESCRIPTION
**Pull request by annanixon**
Add docs for new metric settings

### CODE REVIEW

**Comment by farkmarnum** [comment]
The GIF is 11MB, which is pretty big — could you make it smaller by cropping off the sides and potentially changing the framerate / bitrate? Maybe someone on product or design could help with that if needed.

File: docs/docs/dashboards/Metrics.md [hunk 1]
```diff
@@ -1,26 +1,45 @@
...
+**Adding Filters:** Click the + icon in the "Filters" section to add filters to your metric. These filters will only apply to this metric, and will not be editable by viewers.
+
+**Updating Filters:** Click on individual filter chips in the "Filters" section to edit or delete filters that exist on a metric.
+
+**Ignore Dashboard Filters:** Check this box if you want this metric to ignore
+[dashboard filters](Filters.md). When this is set, only the filters specified on the metric will be used.
+
+## Formatting
+
+![Updating colors](imgs/color.gif)
```
Comment (trucklos) [inline docs/docs/dashboards/Metrics.md:43]: I would consider just removing this one.  Very image heavy and I think people get the idea by the time they get ehre

File: docs/docs/dashboards/Metrics.md [hunk 2]
```diff
@@ -1,26 +1,45 @@
...
+**Sparkline:** Click the toggle to enable or disable a sparkline visualization of your metric. Select a datetime or numeric attribute to plot the metric value against and adjust the granularity or bin size to see trends week to week, month to month, etc.
 
-Check this box if you want this metric to ignore
-[dashboard filters](Filters.md). When this is set, only the filters specified
-on the metric will be used.
+#### Relative Metrics
+
+![Setting up a relative metric](imgs/relative.gif)
+
+**Metric Value:** Choose a [data model](../data-modeling/Data-Models-Overview.md) to select a metric from, then a metric within that data model. Then select the datetime or numeric attribute that you want to aggregate the metric over and adjust the granularity or bin size to see values over the most recent week, month, etc of your dataset.
```
Comment (trucklos) [inline docs/docs/dashboards/Metrics.md:26]: - The first sentence doesn't flow for me.  "choose a data model and metric to display."
- "Select a date and granularity to go on the x axis of the sparkline.  Less commonly used, but it's also possible to choose a number for the x axis"

File: docs/docs/dashboards/Metrics.md [hunk 3]
```diff
@@ -1,26 +1,45 @@
...
+**Metric Value:** Choose a [data model](../data-modeling/Data-Models-Overview.md) to select a metric from, then a metric within that data model to display the value of.
 
-#### Ignore Dashboard Filters
+**Sparkline:** Click the toggle to enable or disable a sparkline visualization of your metric. Select a datetime or numeric attribute to plot the metric value against and adjust the granularity or bin size to see trends week to week, month to month, etc.
 
-Check this box if you want this metric to ignore
-[dashboard filters](Filters.md). When this is set, only the filters specified
-on the metric will be used.
+#### Relative Metrics
+
```
Comment (trucklos) [inline docs/docs/dashboards/Metrics.md:23]: - To me, the actual value for the metric is pretty confusing when interacting with relative metric mode.  I don't change the filters at all and the number is jumping around pretty dramatically.  It's worth calling out very explicitly what the big number is when you're in "relative metric" mode.
- I would have an "info callout" at the top of this section or at least a sentence in the metric value section: "we select the last value in the series".  Maybe even an illustration (I would find more valuable than the gifs tbh)

File: docs/docs/dashboards/Metrics.md [hunk 4]
```diff
@@ -1,26 +1,45 @@
...
+![Setting up a relative metric](imgs/relative.gif)
+
+**Metric Value:** Choose a [data model](../data-modeling/Data-Models-Overview.md) to select a metric from, then a metric within that data model. Then select the datetime or numeric attribute that you want to aggregate the metric over and adjust the granularity or bin size to see values over the most recent week, month, etc of your dataset.
+
+**Sparkline:** Click the toggle to enable or disable a sparkline visualization of your metric. Values will be plotted over the attribute you've selected to calculate the metric.
+
+**Comparison Text:** Click the toggle to enable or disable comparisons. When enabled the tile will display the difference between the value for the most recent period and the prior period in your data. For example, if you have your relative metric set to show user count over a month enabling this feature will show you the difference between users this month and last month.
```
Comment (trucklos) [inline docs/docs/dashboards/Metrics.md:30]: Can remove the first sentence.  Just start with "When comparisons are enabled..."

**Review by trucklos — APPROVED** [review]
This looks good, some copy writing edits proposed.  I agree the gif's add some value,  I don't think we need four.  We sometimes shy away from them because if we change the UI then these will be out of date.

**Pull request merged**